### DISCUSSION TITLE
Are two disks a strict requirement?

### DISCUSSION THREAD

**Discussion by deepakggoel** [cozystack/cozystack#346] [Q&A] [answered]
The quick start says that a server needs to have 2 hdds. Can I use a server that has only 1 hdd?

**Comment by kvaps** [comment]
For now yes, however you can try Talos 1.8.0 it implements disk management.

https://www.talos.dev/v1.8/talos-guides/configuration/disk-management/

So you will be able to utilize extra partition for the data.

**Reply by IvanHunters** [reply, marked as answer]
Here we described how you can install everything on one disk
https://cozystack.io/docs/operations/faq/#how-to-allocate-space-on-system-disk-for-user-storage
